packages/playwright-core/src/server/electron/electron.ts

```diff
@@ -114,6 +120,28 @@ export class ElectronApplication extends SdkObject {
   }
 }
 
+async function urlToNodeWSEndpoint(progress: Progress, endpointURL: string) {
+  if (endpointURL.startsWith('ws'))
+    return endpointURL;
+  progress.log(`<ws preparing> retrieving websocket url from ${endpointURL}`);
+  const httpURL = endpointURL.endsWith('/') ? `${endpointURL}json/` : `${endpointURL}/json/`;
```

Comment:
Please, remove the last `/`.
```suggestion
  const httpURL = endpointURL.endsWith('/') ? `${endpointURL}json` : `${endpointURL}/json`;
```

For some reason, the `json/` endpoint here returns 404: 
<img width="289" alt="image" src="https://user-images.githubusercontent.com/9501115/213597269-f0501efd-fc63-4913-bebb-90528ba48a24.png">

But without the last slash works:
<img width="254" alt="image" src="https://user-images.githubusercontent.com/9501115/213597332-1ea07c36-a25b-485e-bd4e-d84a49acaa1a.png">